docs/4.0/faq.md

```diff
@@ -28,9 +28,18 @@ Yes, Teleport supports reverse SSH tunnels out of the box. To configure
 behind-firewall clusters refer to [Trusted Clusters](admin-guide.md#trusted-clusters) 
 section of the Admin Manual.
 
+### Can individual nodes create reverse tunnels to a proxy server without creating a new cluster? 
+
+This has been a long standing [request](https://github.com/gravitational/teleport/issues/803) of Teleport and
+it has been fixed with Teleport 4.0.   Once you've upgraded your Teleport Cluster and node Client
+change the node config option `--auth-server` to point to the Proxy Server URL. As defined in 
```

Comment:
The proxy actually listens on 3 addresses: 3080 (web), 3023 (proxy SSH), and 3024 (reverse tunnel SSH). We should make it clear it's the web port here.

Maybe something like:

"Once you've upgraded your Teleport Cluster, change the node config option `--auth-server` to point to web proxy address (this would be `public_addr` and `web_listen_addr` in file configuration)."